cmd/gopogh-server/flake_chart.js

```diff
@@ -0,0 +1,597 @@
+// Displays an error message to the UI. Any previous message will be erased.
```

Comment:
Lots of uneven indenting and inconsistent usage of semicolons, I'd recommend running this file through a formatter

Reply:
Done. Not sure whether it fixed everything though